framework/src/modules/chain/submodules/loader.js

```diff
@@ -755,8 +755,14 @@ __private.createSnapshot = height => {
 	}
 
 	const snapshotRound = library.config.loading.snapshotRound;
+	if (!snapshotRound && Number.isNaN(parseInt(snapshotRound))) {
```

Comment:
wouldn't `Number.isNaN(parseInt(snapshotRound))` be `false` for e.g. `-3`?

Reply:
It's not possible to pass this as `commander` considers this as a flag and fails.
But I can add it as an extra check in the code?

Reply:
An extra check or at least a comment would be nice: at the moment the code does not quite correspond to the error message.

Reply:
It throws `error: unknown option `-1'` which is pretty clear I think. Should make you think about not snapshotting for negative numbers

Reply:
Looking at the code it's not so clear (`Number.isNaN(parseInt(snapshotRound))` vs. `'Unable to create snapshot, snapshot parameters should be an integer equal to or greater than zero'`); maybe it is just me though, feel free to ignore.